volume-group-operator/main.go

```diff
@@ -79,18 +85,22 @@ func main() {
 	})
 	exitWithError(err, "unable to start manager")
 
-	log := logr.Logger{}
+	log := ctrl.Log.WithName("controllers").WithName("VolumeGroup")
 	grpcClientInstance, err := getControllerGrpcClient(cfg, log)
 	exitWithError(err, err.Error())
 
+	controllerUtils := utils.ControllerUtils{
```

Comment:
why do we want to do it?

Reply:
so they would have an initiated `Client`

Reply:
but you already do it [here](https://github.com/IBM/ibm-block-csi-operator/blob/d10932232af81a6ff670236a821d15d81fb6d6f7/volume-group-operator/main.go#L97), no?

Reply:
but they are not the same package, do you want to pass the client to each method like the logger?

Reply:
done